tsuru/client/env.go

```diff
@@ -200,28 +243,41 @@ func (c *EnvUnset) Flags() *gnuflag.FlagSet {
 func (c *EnvUnset) Info() *cmd.Info {
 	return &cmd.Info{
 		Name:    "env-unset",
-		Usage:   "env unset <ENVIRONMENT_VARIABLE1> [ENVIRONMENT_VARIABLE2] ... [ENVIRONMENT_VARIABLEN] [-a/--app appname] [--no-restart]",
-		Desc:    `Unset environment variables for an application.`,
+		Usage:   "env unset <ENVIRONMENT_VARIABLE1> [ENVIRONMENT_VARIABLE2] ... [ENVIRONMENT_VARIABLEN] [-a/--app appname] [-j/--job jobname] [--no-restart]",
+		Desc:    `Unset environment variables for an application or job.`,
 		MinArgs: 1,
 	}
 }
 
 func (c *EnvUnset) Run(context *cmd.Context, client *cmd.Client) error {
 	context.RawOutput()
-	appName, err := c.AppName()
+
+	err := checkAppAndJobInputs(c.appName, c.jobName)
 	if err != nil {
 		return err
 	}
+
 	v := url.Values{}
 	for _, e := range context.Args {
 		v.Add("env", e)
 	}
 	v.Set("noRestart", strconv.FormatBool(c.noRestart))
-	u, err := cmd.GetURL(fmt.Sprintf("/apps/%s/env?%s", appName, v.Encode()))
+
+	var path string
```

Comment:
this should be a function since it repeats previously:
```suggestion
func chooseAPIParams(appName, jobName string) (string, string) {
	switch appName {
	case "":
		return fmt.Sprintf("/jobs/%s/env", jobName), "1.13"
	default:
		return fmt.Sprintf("/apps/%s/env", appName), "1.0"
	}
}

path, apiVersion := chooseAPIParams(c.appName, c.jobName)
```

Reply:
ty, I missed this one